Documentation/spelling_wordlist.txt

```diff
@@ -106,6 +106,7 @@ deserialization
 dev
 Devel
 Dinan
+diff
```

Comment:
this seems to be a leftover, but I am ok with that word in our wordlist

Reply:
I had to add that to get the build to pass locally /shrug